getting-started/first-steps.md

```diff
@@ -10,64 +10,52 @@ position: 1
 
 # First Steps
 
-This article explains how to get started with one of the Telerik Document Processing Libraries quickly. As the libraries are UI-Independent and cover all .NET technologies, from desktop and web to mobile, and can also be deployed in client, server-side and cloud apps you need to consider the suite with which you've obtained the product.
+In this getting started guide, we create a simple application that uses the Telerik Document Processing libraries to create a DOCX document and then export it as a PDF file.
 
-Once you have first example up and running, take a look at the [next steps](#next-steps) section to start exploring the rest of the functionality in more details.
-
-For additional resources you can also review the Related Articles section on the right.
-
->You can download the packages that contain Telerik Document Processing from your [Telerik account](https://www.telerik.com/account/my-downloads).
+The Telerik Document Processing libraries that we use in this guide are UI-independent and cover all .NET technologies, from desktop and web to mobile. They can also be deployed in client, server-side and cloud apps.
 
 ## Step 1: Installing on Your Computer
 
-Telerik Document Processing is part of several Telerik bundles and is installed following the steps for installing the suite with which you've obtained the product.
-
-1. UI for ASP.NET AJAX - [Installing Telerik UI for ASP.NET AJAX](http://docs.telerik.com/devtools/aspnet-ajax/installation/which-file-do-i-need-to-install). 
-When the installation completes, the Telerik Document Processing assemblies will be available in the *AdditionalLibraries* sub-folder.
-
-1. UI for ASP.NET MVC - [Installing Telerik UI for ASP.NET MVC](http://docs.telerik.com/kendo-ui/aspnet-mvc/introduction#installation). 
-When the installation completes, the Telerik Document Processing assemblies will be available in the *AdditionalLibraries* sub-folder.
-
-1. UI for ASP.NET Core - [Installing Telerik UI for ASP.NET Core](https://docs.telerik.com/aspnet-core/getting-started/installation/document-processing). 
-The libraries can be used through the available NuGet packages.
+Since we distribute Telerik Document Processing libraries as an addition to several Telerik UI component bundles, chances are that the libraries are already installed on your system. In this case, all you need is to locate them. The table below provides links to the installation instructions for each of the Telerik UI component suites that give you access to the Telerik Document Processing libraries. If the standard installation of your Telerik UI component suite includes the Document Processing assemblies, the table also shows their default location.
```

Comment:
I think we need to add more detail on how the DPL assemblies are distributed with the Telerik UI suites.